### PR TITLE
Fixes in python main function, .gitignore added

### PR DESCRIPTION
**Pull request by jwd83**
Hey there,

Fixed handling of sys.argv in the main function in the python version. The parameters were being passed improperly.

Also when I cloned from GitHub to one of our Ubuntu boxes the windows line endings were causing a problem when trying to execute the python script. Was getting the can't find /usr/bin/python^M error. Converted to UNIX line endings.

Also added a .gitignore to ignore the common file extensions so when working in the cloned repo it's not throwing uncommitted file messages for the .LOG & .csv files.

### CODE REVIEW

File: python/RCP_conv.py [hunk 1]
```diff
@@ -1,123 +1,131 @@
...
+					if current_row[i] == '':
+						current_row[i] = previous_row[i]
+
+				previous_row = current_row
+				writer.writerows([current_row])
+
+
+
+if __name__ == "__main__":
+	# print len(sys.argv)
```
Comment (brentpicasso) [inline python/RCP_conv.py]: commented out code

**Comment by brentpicasso** [comment]
Looks good! just one small thing and I think it's good to go.

**Comment by jwd83** [comment]
Ah, remove the print len?

**Comment by jwd83** [comment]
Added a commit with that.

**Comment by brentpicasso** [comment]
\o/ thanks Jared!

**Comment by vtjballeng** [comment]
Brent, Jared works here at Ballenger Motorsports and got this going today.  Good work Jared! Some notes that should go on the main page.  1 - Be more specific about dlog99 and where to get it either on the page or with a direct link.  2 - on import, a user needs to specify float which will not be obvious to the casual user.  3. after import in dlog99, the stf file should be saved and then opened with gems data analysis package like 4.1.1 or higher.  this step should be spelled out so a user can have a good experience with the modern package.  

other notes
4. Are you able to get a track view working?  I see invalid log source error 5. this data file does not have any lap markers to analyse, so i can't break it down by lap yet.

**Comment by vtjballeng** [comment]
1. track view working.  have to go in and specify gps as the source.

**Pull request merged**